src/processout/applepay.ts

```diff
@@ -137,7 +137,11 @@ module ProcessOut {
             if (!data.success) {
               t.onerror(new Exception(data.error_code, data.message));
               t.session.abort();
-            } else t.onsuccess(data.card);
+            } else{
```

Comment:
nitpick: TS/JS formatter not kicking in?

Reply:
we don't have formatter set up at all :D If you would run pretter on this, commit would be +5k/-5k :P

Reply:
yeah, so i see on the other file :D :this-is-fine: